apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -148,12 +148,17 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     new_exits_kv_pairs =
       new_exits
       |> Enum.zip(exit_contract_statuses)
-      |> Enum.map(fn {%{eth_height: eth_height}, {address, token, amount, utxo_pos} = contract_status} ->
-        {Utxo.Position.decode(utxo_pos),
+      |> Enum.map(fn {event, contract_status} ->
+        %{eth_height: eth_height, call_data: %{utxo_pos: utxo_pos, output_tx: tx_bytes}} = event
```

Comment:
1. `event` seems to be use just here - consider destructure it in params
1. it might be just Watcher's perspective, but please rename `tx_bytes -> txbytes`

Reply:
1. if I pattern-match the fields of `event` in the `fn` params the line becomes too long, but the formatter can't cope with that
2. :+1:

Reply:
2. I'll rename only here to avoid an all-file diff